include/api/ElasticsearchStateIndex.h

```diff
@@ -11,13 +11,11 @@
 namespace ml {
 namespace api {
 //! Elasticsearch index for state
-static const std::string ML_STATE_INDEX(".ml-state");
-static const std::string MODEL_STATE_TYPE("model_state");
-static const std::string REGRESSION_TRAIN_STATE_TYPE("predictive_model_train_state");
+extern const std::string ML_STATE_INDEX;
```

Comment:
Windows link errors are caused by these needing `API_EXPORT`.
```suggestion
API_EXPORT
extern const std::string ML_STATE_INDEX;
```